spacetemplate/assets/agile.yaml

```diff
@@ -218,8 +224,10 @@ work_item_types:
   extends: *agileCommonTypeID
   name: Task
   can_construct: yes
-  description: 
-    An atomic task on the project.
+  description:
+    Work assigned to team members to implement a story. They are generally
```

Comment:
I think this should not be part of the description:  *They are generally
    measured in units of 0.5 days, for example, four hours, eight hours, sixteen
    hours.*

Reply:
@kwk this infotip was as per inputs and reviews from Todd. Since it mentions that it is 'generally measured', it is not obligatory that folks follow this. I am open to suggestions, maybe we could check with the team and drop it if most folks feel its not required.